Tests/BrokerServiceTests/UnitTest1.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace BrokerServiceTests
```

Comment:
Strange to see this in _create operation service tests_

Reply:
> Strange to see this in _create operation service tests_

it was added back in the previous request, in this I didn’t add it specifically